bundler/spec/runtime/platform_spec.rb

```diff
@@ -434,11 +434,11 @@
       G
 
       expect(out).to include("lockfile does not have all gems needed for the current platform")
-      expect(the_bundle).to include_gem "platform_specific 1.0 x64-mingw32"
+      expect(the_bundle).to include_gem "platform_specific 1.0 x64-mingw-ucrt"
     end
   end
 
-  %w[x86-mswin32 x64-mswin64 x86-mingw32 x64-mingw32 x64-mingw-ucrt aarch64-mingw-ucrt].each do |platform|
+  %w[x86-mswin32 x64-mswin64 x86-mingw32 x64-mingw-ucrt x64-mingw-ucrt aarch64-mingw-ucrt].each do |platform|
```

Comment:
Proposing cleaning up this duplicate `x64-mingw-ucrt` oversight on my part over here.
- https://github.com/rubygems/rubygems/pull/8736